runtime/convertValues.go

```diff
@@ -256,36 +266,118 @@ func exportCompositeValue(
 	)
 }
 
-func exportDictionaryValue(
-	v *interpreter.DictionaryValue,
+func exportSimpleCompositeValue(
```

Comment:
Exporting simple composite values is pretty much the same code as exporting normal/user-defined composite values